src/modules/navigator/navigator_main.cpp

```diff
@@ -662,18 +662,6 @@ Navigator::run()
 			_navigation_mode_array[i]->run(_navigation_mode == _navigation_mode_array[i]);
 		}
 
```

Comment:
What was the rationale to remove this? And what is the test coverage? I can see that this could make sense, but I'd like to clarify what the design or test coverage was.

Reply:
@RomanBapst Could you also have a look and chime in?

Reply:
It's this one pending fixedwing tests: https://github.com/PX4/Firmware/pull/14134

Reply:
Nevermind, I can't remove it. It's needed, otherwise the tests fail.

Reply:
This change has been merged. And this PR is now rebased.